src/service/resources-impl.js

```diff
@@ -742,7 +742,7 @@ export class Resources {
         if (diff == 0) {
```

Comment:
yes I can do that. But I think the point of having a check for `diff==0` here is to separate this case with all the checks below. 
Another thing that confuse me is in `#scheduleChangeSize` we reject the size change only when `newWidth=layoutBox.width && newHeight=layoutBox.height` But why do we reject the size change as long as `newHeight=layoutBox.height`